2_gitlab/versions.tf

```diff
@@ -19,9 +19,9 @@ terraform {
 }
 
 provider "google" {
-  version = "= 3.28.0"
+  version = "~> 3.28.0"
```

Comment:
Did you want to make this 3.29 like the other locations?

Reply:
There was an accidental breaking change in the 3.29.0 provider, so I did most of the GitLab testing with 3.28.0. For the time being I'd prefer to stay as-is and bump them to be consistent in a future PR.